docs/t-sql/statements/set-transaction-isolation-level-transact-sql.md

```diff
@@ -75,7 +75,10 @@ SET TRANSACTION ISOLATION LEVEL READ UNCOMMITTED
   
 -   If READ_COMMITTED_SNAPSHOT is set to OFF (the default), the [!INCLUDE[ssDE](../../includes/ssde-md.md)] uses shared locks to prevent other transactions from modifying rows while the current transaction is running a read operation. The shared locks also block the statement from reading rows modified by other transactions until the other transaction is completed. The shared lock type determines when it will be released. Row locks are released before the next row is processed. Page locks are released when the next page is read, and table locks are released when the statement finishes.  
   
--   If READ_COMMITTED_SNAPSHOT is set to ON, the [!INCLUDE[ssDE](../../includes/ssde-md.md)] uses row versioning to present each statement with a transactionally consistent snapshot of the data as it existed at the start of the statement. Locks are not used to protect the data from updates by other transactions.  
+-   If READ_COMMITTED_SNAPSHOT is set to ON, the [!INCLUDE[ssDE](../../includes/ssde-md.md)] uses row versioning to present each statement with a transactionally consistent snapshot of the data as it existed at the start of the statement. Locks are not used to protect the data from updates by other transactions.
+
+> [!IMPORTANT]  
+> Choosing a transaction isolation level does not affect the locks acquired to protect data modifications. A transaction always gets an exclusive lock on any data it modifies, and holds that lock until the transaction completes, regardless of the isolation level set for that transaction. Additionally, an update made at the READ_COMMITTED isolation level usse update locks on the data rows selected. By contast, if an update made  at the SNAPSHOT isolation level uses row versions to select rows to update. For read operations, transaction isolation levels primarily define the level of protection from the effects of modifications made by other transactions. See the [Transaction Locking and Row Versioning Guide](https://docs.microsoft.com/en-us/sql/relational-databases/sql-server-transaction-locking-and-row-versioning-guide) for more information.

```

Comment:
The third sentence misspells uses (usse). Please fix.
The fourth sentence is incomplete as written. I think you mean to say "By contast, an update made at the SNAPSHOT isolation level uses row versions to select rows to update." Please confirm and fix.
Also, you use 2 spaces in a few places rather than one.

Reply:
Thank you for making this submission. I am asking you to make the change to ensure you get credit for the PR.